02-social-links/style.css

```diff
@@ -0,0 +1,223 @@
+* {
+  margin: 0;
+  padding: 0;
+  /* box-sizing: content-box; */
+  font-family: "Inter", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-style: normal;
+  font-variation-settings: "slnt" 0;
+  transition: all 0.2s;
+}
+article {
+  display: flex;
+}
+a {
+  text-decoration: none;
+  color: inherit;
+  font-weight: 700;
+  font-size: 0.9rem;
+}
+.logo {
+  display: block;
+  width: 100px;
+  border-radius: 50%;
+  margin: 0.5rem auto;
+  margin-bottom: 1.5rem;
+}
+li {
+  list-style: none;
+  background-color: hsl(0, 0%, 20%);
+  color: hsl(0, 0%, 100%);
+  padding: 0.8rem 2.5rem;
+  border-radius: 0.5rem;
+  width: 15rem;
```

Comment:
It's better not to hardcode width with absolute units. Better use relative units. `%` in this specific case. Also, the default value for li's width is already 100%. So, you should not set it at all here.